### PR TITLE
feat(perf): Add avg to Txn Duration Breakdown widget

### PR DESCRIPTION
**Pull request by mjq-sentry**
We've had a number of requests internally and externally to add `avg()` to the Transaction Summary page's Duration Breakdown widget, alongside the existing percentile options. Add `avg()` as an option, but make it unselected by default.

Unlike the percentile functions, `avg()` doesn't have `transaction.duration` as its default argument: it must be passed explicitly. Updated the widget to pass this argument by default, but strip it back out before the results are displayed to avoid cluttering the widget.

Default:
Screenshot 2023-07-19 at 1 49 56 PM

With only `avg()` and `p50()` toggled on:
Screenshot 2023-07-19 at 1 51 17 PM

### CODE REVIEW

File: static/app/views/performance/transactionSummary/transactionOverview/durationChart/index.tsx [hunk 1]
```diff
@@ -142,16 +144,24 @@ function DurationChart({
...
-            timeFrame={timeFrame}
-            {...contentCommonProps}
-          />
-        )}
+        {({results, errored, loading, reloading, timeframe: timeFrame}) => {
+          // Function arguments are superflous and take up space in the chart
+          // labels, so remove them.
+          const series = results?.map(result => ({
+            ...result,
+            seriesName: `${parseFunction(result.seriesName)?.name}()`,
```
Comment (k-fish) [inline static/app/views/performance/transactionSummary/transactionOverview/durationChart/index.tsx:152]: I sometimes just make a named function to explain the `why` for something like this. eg. `function stripSeriesForLegend()`

**Review by k-fish — APPROVED** [review]
Nice Matt 👍

**Comment by k-fish** [comment]
Also, I'd maybe check `p100` vs `avg` color, you might need to pick a larger palette for the legend.

**Comment by mjq-sentry** [comment]
> Also, I'd maybe check `p100` vs `avg` color, you might need to pick a larger palette for the legend.

Screenshot 2023-07-19 at 3 34 35 PM

I think it looks okay - @k-fish what do you think?

File: static/app/views/performance/transactionSummary/transactionOverview/durationChart/index.tsx (hunk 1 above)
Comment (mjq-sentry) [reply static/app/views/performance/transactionSummary/transactionOverview/durationChart/index.tsx]: I like it :+1: Changed!

**Comment by k-fish** [comment]
Looks fine, nvm, I'm guessing the palette was picked based on array length, perfect 👍

**Pull request merged**